### DISCUSSION TITLE
Overlapping Plots (line)

### DISCUSSION THREAD

**Discussion by gkb999** [pydata/xarray#7886] [General] [answered]
Hi all, 
I know mine is a very basic question.
But, I'm figuring out a way to plot overlapping plots from my daily data, something like:
![image](https://github.com/pydata/xarray/assets/7091088/c4a77221-0f73-4906-a0e5-e65bbe052b92)

where days of the year are on x-axis and y-axis - the variable name.
All the eyars need to overlap one over other, rather than a continuous one like:
Is there a way to do without converting the xarray_dataset to pandas data frame individually?

My Dataset format:
```python
<xarray.DataArray 'z' (time: 3283)>
array([ 11.76674441,  13.66537748,  15.64659361, ..., -15.93535297,
       -14.4851825 , -16.47597701])
Coordinates:
  * time     (time) object 2013-01-01 00:00:00 ... 2021-12-31 00:00:00
```
The data is a daily data for 9 years (with few missing days)

**Comment by TomNicholas** [comment, marked as answer]
I'm a bit unclear on exactly what you're trying to do, but have you tried using the `hue` keyword argument to the `.plot.line()` method? It allows you to make plots like this - [documentation here](https://docs.xarray.dev/en/stable/user-guide/plotting.html#multiple-lines-showing-variation-along-a-dimension)

![image](https://github.com/pydata/xarray/assets/35968931/08180bca-0867-48aa-8e08-d34b7ec180eb)
